sdk/nodejs/cmd/run/run.ts

```diff
@@ -53,14 +64,38 @@ function projectRootFromProgramPath(program: string): string {
     }
 }
 
+async function npmPackageRootFromProgramPath(programPath: string): Promise<string> {
+    // pkg-dir is an ESM module which we use to find the location of package.json
+    // Because it's an ESM module, we cannot import it directly.
+    const { packageDirectory } = await dynamicImport("pkg-dir");
+    // Check if programPath is a directory. If not, then we
+    // look at it's parent dir for the package root.
+    let isDirectory = false;
+    if(fs.existsSync(programPath)) {
```

Comment:
minor: Mind running the formatter over this? `if(` here versus `if (` below.

Reply:
Am i losing my mind? I'm not seeing a formatted or formatter config in this directory.
We don't have a `make format` rule for JavaScript, nor is `prettier` installed as a dev dependency. The `make format` rule just runs gofmt. 😕 ❓ 

I can open an issue to add a formatter to the repo.

For now I just added the space, which looks consistent with other locations in the file. 🤷🏻

Reply:
https://github.com/pulumi/pulumi/issues/12781